### PR TITLE
emulationstation: 2.0.1a -> 2.11.2

### PR DESCRIPTION
**Pull request by soyouzpanda**
## Description of changes

Updated emulationstation package from Aloshi's emulationstation (unmaintained) to RetroPie's emulationstation (lastest stable release)

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [x] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by kirillrdy** [comment]
@soyouzpanda greetings !
please sqush commits and change commit message to match https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md

also update PR title to match commit message

**Comment by soyouzpanda** [comment]
I've made the changes

PS: It was my first contribution, so thanks for the tips :)

**Comment by kirillrdy** [comment]
> I've made the changes
> 
> PS: It was my first contribution, so thanks for the tips :)

thank you for your first contribution !
there is still a merge commit that should be removed from this PR, let me know if you need assistant to fix that.

File: pkgs/applications/emulators/emulationstation/default.nix [hunk 1]
```diff
@@ -1,31 +1,21 @@
 { lib, stdenv, fetchFromGitHub, pkg-config, cmake, curl, boost, eigen
-, freeimage, freetype, libGLU, libGL, SDL2, alsa-lib, libarchive
+, freeimage, freetype, libGLU, libGL, rapidjson, SDL2, alsa-lib, vlc
 , fetchpatch }:
```
Comment (kirillrdy) [inline pkgs/applications/emulators/emulationstation/default.nix:3]: fetchpatch is unused

**Comment by kirillrdy** [comment]
Result of `nixpkgs-review pr 261276` run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>emulationstation</li>
  </ul>
</details>

**Comment by soyouzpanda** [comment]
> > I've made the changes
> > PS: It was my first contribution, so thanks for the tips :)
> 
> thank you for your first contribution ! there is still a merge commit that should be removed from this PR, let me know if you need assistance to fix that.

Ahah, I was wondering if I should keep it or not :/ I deleted the merge commit. Thanks again :)

File: pkgs/applications/emulators/emulationstation/default.nix (hunk 1 above)
Comment (soyouzpanda) [reply pkgs/applications/emulators/emulationstation/default.nix]: Fixed.

**Comment by kirillrdy** [comment]
@soyouzpanda any suggestion on how to test this ?

```
[nix-shell:~/.cache/nixpkgs-review/pr-261276-3]$ ./results/emulationstation/bin/emulationstation 
Oct 16 21:01:34 lvl0: 	Error - File type unknown!
Segmentation fault (core dumped)

```

**Comment by Janik-Haag** [comment]
Result of `nixpkgs-review pr 261276` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>emulationstation</li>
  </ul>
</details>

**Comment by Janik-Haag** [comment]
image
the changes look good but I'm also experiencing a core dump :(

feel free to re-request review once that's fixed.

**Comment by soyouzpanda** [comment]
Ok, so it should work as expected now. I cannot run `nixpkgs-review` to confirm it because it keeps crashing...

**Review by Janik-Haag — APPROVED** [review]
lgtm

**Review by kirillrdy — APPROVED** [review]

**Pull request merged**